qualification/antenna_channelised_voltage/test_delay.py

```diff
@@ -532,6 +532,7 @@ async def test_delay_phase(
 
 
 @pytest.mark.requirements("CBF-REQ-0128,CBF-REQ-0112")
+@pytest.mark.xfail(reason="requirements unclear")
```

Comment:
My only concern is that, as far as I understand things, this marks all the `phase_rate` tests as `xfail`.

My reading of the [pytest documentation ](https://docs.pytest.org/en/6.2.x/skipping.html) suggests that something which unexpectedly passes should be marked as `xpass`, but that doesn't seem to be the case with the L-band phase-rate test in your report.

All this looks fine to me, I don't have any objections, but I'm struggling to understand why this is happening.

Reply:
It's reported as xpass in the summary on stdout, but the `outcome` field in the JSON structure is still `passed`. There is probably some way to distinguish an xpass from a normal pass, but I didn't see much need to distinguish them.

Reply:
I see. So there's a bit of a disconnect somewhere.

This is fine for now I think.